server/remote/msgraph/get_me.go

```diff
@@ -3,13 +3,31 @@
 
 package msgraph
 
-import "github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
 
 func (c *client) GetMe() (*remote.User, error) {
 	graphUser, err := c.rbuilder.Me().Request().Get(c.ctx)
 	if err != nil {
 		return nil, err
 	}
+
+	if graphUser.ID == nil {
+		return nil, fmt.Errorf("User has no ID")
```

Comment:
Do you mind using `errors.New` since we are not injecting any values?

```suggestion
		return nil, errors.New("User has no ID")
```